docs/discovery.md

```diff
@@ -180,6 +186,19 @@ You can list the nodes in the cluster.
     swarm list zk://<zookeeper_addr1>,<zookeeper_addr2>/<optional path prefix>
     <node_ip:2375>
 
+#### Use TLS with Distributed K/V discovery
+
+You can securely talk to the distributed k/v store using TLS.
+This works only for Consul and etcd, SSL support for zookeeper is coming.
+
+To connect securely to the store, make sure you generate the certificates accordingly
+and use (taking Consul as an example):
+
+```
+swarm join --advertise=<node_ip:2375> --discovery-opt kv.cacertfile=/path/to/mycacert.pem --discovery-opt kv.certfile=/path/to/mycert.pem --discovery-opt kv.keyfile=/path/to/mykey.pem consul://<consul_addr>/<optional path prefix>
```

Comment:
For the documentation, I'd wrap this, to prevent people having to scroll to see the entire command, e.g.

```bash
$ swarm join \
  --advertise=<node_ip:2375> \
  --discovery-opt kv.cacertfile=/path/to/mycacert.pem \
  --discovery-opt kv.certfile=/path/to/mycert.pem \
  --discovery-opt kv.keyfile=/path/to/mykey.pem \
  consul://<consul_addr>/<optional path prefix>
```